docs/requirements.txt

```diff
@@ -0,0 +1,2 @@
+Sphinx==1.7.5
```

Comment:
Please add it in `requirements/dev.txt`
So, now the directory structure would be `requirements/`(which is a folder), `requirements/common.txt`, `requirements/dev.txt`, `requirements/prod.txt`.

Reply:
Since the production settings are already mentioned at `setup.py` I don't think we need a `prod.txt`. And these settings are only needed for working on the docs, so is it right to put it in the `dev.txt`?